docs/introduction/architecture.md

```diff
@@ -19,103 +19,106 @@ need to implement Server-Sent Event (SSE) connections fully.
 
 - **Well Tested**: By relying on one core library making decisions across our SDKs and APIs, 
 we can more easily ensure it is well-tested and reliable. In addition to thorough unit testing, 
 we have a cross-platform end-to-end SDK test-harness to ensure platform consistency.
 
-- **Secure**: WASM runs in a [memory-safe sandboxed execution environment](https://webassembly.org/docs/security/) 
-that has been proven to be secure over many years.
+- **Secure**: WASM runs in a [memory-safe sandboxed execution environment](https://webassembly.org/docs/security/)
+that has proven its security credentials across web browsers.
 
-However, WASM is not a silver bullet, and for certain very highly threaded low-level use cases, 
+However, WASM is not a silver bullet, and for certain very highly threaded lower-level use cases, 
 we have built a native implementation, for example, in our [GO SDK](https://github.com/DevCycleHQ/go-server-sdk).
 
 ## Local Bucketing Server SDK Architecture
 
 ![Architecture Docs Diagrams - Local Server SDK.svg](/architecture-docs-diagrams-local-server-sdk.svg)
 
-**Note: Server SDKs currently do not support Server-Sent Events for updates to the config.**
+**Note: Server-sent event support for Server SDKs is currently under development.**
 
-1. On initialization, the Server SDK retrieves the configuration data from the CDN, and stores it locally.
+1. On initialization, the Server SDK retrieves the configuration data from the CDN and stores it locally.
 
-2. On each `variableValue()` / `variable()` call, bucketing and segmentation library combines user data, device data, 
-and the configuration data locally to bucket users into features and variations to determine variable values.
+2. On each `variableValue()` / `variable()` call, the bucketing and segmentation library combines user data, device data, 
+and configuration data locally to bucket users into features and variations to determine variable values.
 
-3. Configuration updates are received via polling against the CDN.
+3. Configuration updates are received via polling against the CDN or through a real-time server-sent event (SSE) connection.
 
-4. Event data is aggregated and sent to the Events API on an interval.
+4. Event data is aggregated within the SDK and sent to the Events API on an interval.
 
 ## Cloud Bucketing Server SDK Architecture
 
 ![Architecture Docs Diagrams - Cloud Server SDK.svg](/architecture-docs-diagrams-cloud-server-sdk.svg)
 
-**Note: Server SDKs currently do not support Server-Sent Events for updates to the config.**
+For most use cases, local bucketing SDKs provide superior performance and reliability.
+However, the cloud-bucketing SDKs can make integration easier for specific use cases where access to
+[EdgeDB](/extras/edgedb) to integrate user data between client-side and backend applications is needed.
+
+**Note: Server-sent event support for Server SDKs is currently under development.**
```

Comment:
whys this note repeated in this section? server sent events arent relevant for cloud bucketing anyway